### ISSUE TITLE
Tensorflow in Azure ML

### ISSUE THREAD

**Issue by tanayz** [tensorflow/tensorflow#4256]
I've so far seen people using tensorflow in Azure using in this [link](http://www.mikelanzetta.com/tensorflow-on-azure-using-docker.html).
Also using the advantage of ubuntu in windows tensorflow can be run on
windows pc as well.Here is the [link](http://www.hanselman.com/blog/PlayingWithTensorFlowOnWindows.aspx).
However during a conversation with Windows Azure engineer Hai Ning it came out
that "Azure ML PaaS VMs use Windows OS; TensorFlow is not supported on Windows as of now."
Hence there is no direct way of running tensorflow in Azure ML.
Is there any work around anyone figured out that allows running tensorflow in Azure ML.

NOTE: Only file GitHub issues for bugs and feature requests.  All other topics will be closed.

For general support from the community, see [StackOverflow](https://stackoverflow.com/questions/tagged/tensorflow).
To make bugs and feature requests more easy to find and organize, we close issues that are deemed
out of scope for GitHub Issues and point people to StackOverflow.

For bugs or installation issues, please provide the following information.
The more information you provide, the more easily we will be able to offer
help and advice.
### Environment info

Operating System:

Installed version of CUDA and cuDNN: 
(please attach the output of `ls -l /path/to/cuda/lib/libcud*`):

If installed from binary pip package, provide:
1. A link to the pip package you installed:
2. The output from `python -c "import tensorflow; print(tensorflow.__version__)"`.

If installed from source, provide 
1. The commit hash (`git rev-parse HEAD`)
2. The output of `bazel version`
### Logs or other output that would be helpful

(If logs are large, please upload as attachment or provide link).

**Comment by aselle** [comment]
Others probably have more insight on this, but there is a windows version forthcoming (see #17).

**Comment by mrry** [comment]
Just a small observation: [Azure ML](https://azure.microsoft.com/en-us/services/machine-learning/) is a fully managed service. Even when TensorFlow runs on Windows, you'd have to ask the people who manage that service to add TensorFlow to the set of modules that Azure ML exports.

If you want a managed cloud service that does offer TensorFlow support, the [Google Cloud ML](https://cloud.google.com/ml/) product is in limited preview, and you can [apply](https://services.google.com/fb/forms/machinelearningpreview/) to join the alpha.

**Comment by tobegit3hub** [comment]
Agree with @mrry . Google Cloud ML is the better choice for managed TensorFlow service or you can implement the similar one in bare servers with TensorFlow. 

Azure ML is not the open-source project and I'm not sure if TensorFlow community could help. Maybe asking Microsoft to support TensorFlow could be more helpful.

**Comment by drcrook1** [comment]
This is an interesting thread.  @tanayz can you expand on what you plan or wish to do?

**Comment by aselle** [comment]
You might want to take a look at this new blog post
https://blogs.msdn.microsoft.com/uk_faculty_connection/2016/09/26/tensorflow-on-docker-with-microsoft-azure/https://blogs.msdn.microsoft.com/uk_faculty_connection/2016/09/26/tensorflow-on-docker-with-microsoft-azure/

**Comment by tanayz** [comment]
Thanks @aselle seems like accessing it from AML is doable.However I already found that keras can run in AML,hence keeping this in back burner for now.

**Comment by aselle** [comment]
Automatically closing due to lack of recent activity. Please file a new issue referring to this issue, if the issue is still relevant and important. If this is documenting some kind of bug, be sure to use the latest version of TensorFlow. Thank you so much.

**Comment by prashanthmadi** [comment]
Here is another blog to run tensorflow in Azure App Services 
https://prmadi.com/playing-piano-python-app-in-azure-app-services-linux/
